CRVResponse/src/CrvPhotonGenerator_module.cc

```diff
@@ -222,7 +222,6 @@ namespace mu2e
       filedirs.insert( std::filesystem::path(filespec).parent_path() );
       photonMaker->LoadLookupTable(filespec,_debug);
       photonMaker->SetScintillationYield(_scintillationYields[i]);
-      if(_debug>0) std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<" with scintillation yield of "<<_scintillationYields[i]<<" photons/MeV"<<std::endl;
```

Comment:
This line should have stayed.

Reply:
It looks like Ray has addressed this now.  When you agree that this is ready, please approve the PR.